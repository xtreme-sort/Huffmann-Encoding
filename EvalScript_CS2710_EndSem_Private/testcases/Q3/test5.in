GenerateTreeEnhanced
10
d 2
0 7
m 8
v 18
3 27
u 34
L 42
y 42
F 44
B 47
Quit
