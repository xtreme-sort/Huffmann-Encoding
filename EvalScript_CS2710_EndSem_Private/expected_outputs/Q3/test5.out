((B,(3,u)),((((m,(d,0)),v),L),(y,F)));
